projects/core/components/notification/notification.component.ts

```diff
@@ -53,9 +54,16 @@ export class TuiNotificationComponent {
         @Inject(TUI_CLOSE_WORD) readonly closeWord$: Observable<string>,
         @Inject(TUI_COMMON_ICONS) readonly icons: TuiCommonIcons,
         @Inject(TUI_NOTIFICATION_OPTIONS) readonly options: TuiNotificationDefaultOptions,
+        @Inject(ElementRef) private readonly elementRef: ElementRef<HTMLElement>,
     ) {}
 
     get hasClose(): boolean {
         return !this.hideClose && tuiIsObserved(this.close);
     }
+
+    get isInteractive(): boolean {
```

Comment:
Let's just hide it with CSS, we already have rules matching `a` and `button` inside this component styles.